herddb-core/src/main/java/herddb/model/ColumnTypes.java

```diff
@@ -60,11 +60,11 @@ public static String typeToString(int type) {
             case BOOLEAN:
                 return "boolean";
             case NOTNULL_STRING:
-                return "string not null";
+                return "string";
```

Comment:
So here and for the other not null rypes we have to return 'string not null'

Cc @diegosalvi